src/pages/Account/RequestResetPassword/RequestResetPassword.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ChangeEvent, FormEvent, useState } from 'react';
+
+import { Link } from 'react-router-dom';
+
+import requestResetPassword from '@/apis/requestResetPassword';
+import { MESSAGES, STATUS_CODES } from '@/constants/constantValues';
+import { emailValidationPattern } from '@/helpers/validators';
+
+import type { IAccountInfo } from '../types';
+
+export default function RequestResetPassword(): JSX.Element {
+  const [email, setEmail] = useState<string>('');
+  const [emailValidationMessage, setEmailValidationMessage] = useState<string>('');
+  const [resetPasswordInfo, setResetPasswordInfo] = useState<IAccountInfo>({
+    code: STATUS_CODES.OK,
+    message: '',
+  });
+
+  const handleEmail = (event: ChangeEvent<HTMLInputElement>): void => {
+    setEmailValidationMessage('');
+    setEmail(event.target.value);
+  };
+
+  const validateEmail = (email: string): boolean => {
+    if (!email) {
+      setEmailValidationMessage(MESSAGES.EMAIL_REQUIRED);
+
+      return false;
+    }
+
+    if (!emailValidationPattern.test(email)) {
+      setEmailValidationMessage(MESSAGES.EMAIL_MUST_BE_VALID);
+
+      return false;
+    }
+
+    return true;
+  };
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>): void => {
+    event.preventDefault();
```

Comment:
Should we have a logic for `submitting` state? What will happen if the user clicks on the submit button multiple times?